app/Http/Controllers/UserListController.php

```diff
@@ -91,7 +91,7 @@ public function add_game(Request $request, $listid, $gameid){
         return \Redirect::action('GameController@show', $gameid);
     }
 
-    public function show($userid, $listid){
+    public function show($userid, $listid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->